src/main/java/me/lucko/lpcookbook/listener/PlayerFirstJoinListener.java

```diff
@@ -32,7 +32,7 @@ private void onDataSave(PlayerDataSaveEvent e) {
         Set<Outcome> outcomes = result.getOutcomes();
 
         if (outcomes.contains(Outcome.CLEAN_INSERT)) {
-            Bukkit.broadcastMessage(ChatColor.GREEN + e.getUsername() + " joined the network for the first time!");
+            Bukkit.broadcastMessage(ChatColor.GREEN + e.getUsername() + " joined the server for the first time!"); // made this server instead of network since all the other messages also says server
```

Comment:
Could you revert this change?

`network` is significant here -- this event will not fire when the player joins the _server_ for the first time. It is called when they first join any server in the network.

Reply:
fixed